target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -426,6 +426,16 @@ define Device/tplink_eap245-v3
 endef
 TARGET_DEVICES += tplink_eap245-v3
 
+define Device/tplink_ec230-g1
+  $(Device/tplink-v2)
+  SOC := qca9563
+  IMAGE_SIZE := 16000k
+  DEVICE_MODEL := EC230-G1
+  TPLINK_BOARD_ID := EC230-G1
```

Comment:
That's interesting. I thought TPLINK_BOARD_ID is only required for tp-link-safeloader, which we don't need here since we can't flash? So, can this be removed, or is it needed anywhere else I don't think of?

Reply:
You're right, It's not needed. Sorry I referenced it from other board definition, and didn't know what those variables meant.